crates/uv-toolchain/src/find.rs

```diff
@@ -0,0 +1,13 @@
+use std::path::{Path, PathBuf};
+
+use once_cell::sync::Lazy;
+
+pub static TOOLCHAIN_DIRECTORY: Lazy<PathBuf> = Lazy::new(|| {
```

Comment:
This is unrelated but was annoying to remove due to crate dependency changes.

This module will be used for existing toolchain lookups in the next pull request.